### PR TITLE
rds.py - added more attributes to the returned output

### PR DESCRIPTION
**Pull request by Constantin07**
##### SUMMARY
This PR enriches the output returned by module and adds more properties.
It doesn't change the structure of old returned values so is backwards compatible.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/rds.py

##### ANSIBLE VERSION
```
ansible 2.4.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]
```

##### ADDITIONAL INFORMATION
None.

An example of new returned output:
```
TASK [rds_output : debug] *********************************************
ok: [localhost] => {
    "changed": false,
    "rds_res": {
        "changed": false,
        "instance": {
            "allocated_storage": 100,
            "auto_minor_version_upgrade": false,
            "availability_zone": "eu-west-1a",
            "backup_retention": 14,
            "backup_window": "03:00-03:30",
            "character_set_name": "AL32UTF8",
            "create_time": 1489079017.009,
            "db_name": "ASERTG",
            "db_subnet_groups": {
                "description": "Subnets for the RDS SQL DB Instance",
                "name": "samplesubnetgrouprds-j6paiqkxqp4z",
                "status": "complete",
                "subnets": [
                    {
                        "availability_zone": {
                            "name": "eu-west-1b",
                            "provisioned_iops_capable": false
                        },
                        "identifier": "subnet-3fdba63e",
                        "status": "active"
                    },
                    {
                        "availability_zone": {
                            "name": "eu-west-1a",
                            "provisioned_iops_capable": false
                        },
                        "identifier": "subnet-1ce85f4d",
                        "status": "active"
                    }
                ],
                "vpc_id": "vpc-11edfe4a"
            },
            "endpoint": "test.1fand5a7du9b.eu-west-1.rds.amazonaws.com",
            "engine": "oracle-se",
            "engine_version": "11.2.0.4.v6",
            "id": "test",
            "instance_type": "db.m4.xlarge",
            "iops": null,
            "latest_restorable_time": 1489707802.0,
            "license_model": "bring-your-own-license",
            "maintenance_window": "sat:01:00-sat:01:30",
            "multi_zone": true,
            "option_groups": [
                {
                    "option_group_name": "default:oracle-se-11-2",
                    "status": "in-sync"
                }
            ],
            "parameter_groups": [
                {
                    "parameter_apply_status": "in-sync",
                    "parameter_group_name": "testawsrpprodb01spfile-1ujg7nrs7sgyz"
                }
            ],
            "pending_modified_values": {
                "db_instance_class": null,
                "db_instance_identifier": null,
                "allocated_storage": null,
                "backup_retention_period": null,
                "engine_version": null,
                "iops": null,
                "master_user_password": null,
                "multi_az": null,
                "port": null
            },
            "port": 1530,
            "publicly_accessible": false,
            "read_replica_source_dbinstance_identifier": null,
            "replication_source": null,
            "secondary_avaialbility_zone": "eu-west-1b",
            "status": "available",
            "username": "ADMIN",
            "vpc_security_groups": "sg-daf9b5a3"
        }
    }
}
```

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/rds.py [hunk 1]
```diff
@@ -584,16 +584,69 @@ def get_data(self):
         d = {
             'id': self.name,
             'create_time': self.instance['InstanceCreateTime'],
+            'engine': self.instance['Engine'],
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/rds.py:804]: Can you add [RETURN docs](https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#return-block) please? There is also a pull request out to [separate rds functionality](https://github.com/ansible/ansible/pull/25646); if there is return info you think should be added feel free to leave some feedback.

**Review by s-hertel — CHANGES_REQUESTED** [review]

**Comment by Constantin07** [comment]
@s-hertel Done.

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
shipit

**Comment by pilou-** [comment]
ping: this pull-request seems to be eligible to the `automerge` workflow: adding a comment in order to trigger the bot which should re-run the shippable CI and then remove the `stale_ci` label.

**Pull request merged**